sfa_dash/templates/data/time_widgets.html

```diff
@@ -15,11 +15,13 @@
 <div class="form-element full-width">
   <button type="submit" form="plot-range-adjust" value="Submit" class="btn btn-primary">Update graph</button>
   <form action="" method="post" id="download-form" onsubmit="ParseStartEnd()">
+  {% if is_allowed('read_values') %}
```

Comment:
is there any point in adding any of the time widgets if the user can't read values and I assume there is no/an empty graph?